pkg/batch/executor_test.go

```diff
@@ -78,7 +78,7 @@ func testReadAfterWrite(t *testing.T) {
 	read2Done := make(chan bool)
 	read2Batched := make(chan struct{})
 
-	// we pass a custom delay func that ensures we make the write only after
+	// we pass a custom delay func that ensures we make to write only after
```

Comment:
I don't understand the sentence.  Perhaps "wait" not "make"?

Reply:
Update the comment to reflect the right behavior